### PR TITLE
Show User ID on Account page

### PR DESCRIPTION
**Pull request by filiptronicek**
## Description

Surfaces the user's ID through the Account settings in the Dashboard.

image

## Related Issue(s)

Falls under https://github.com/gitpod-io/gitpod/issues/15654

## How to test

Open the Dashboard in the preview environment :)

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - ft-user-idf0721f5345</li>
	<li><b>🔗 URL</b> - <a href="https://ft-user-idf0721f5345.preview.gitpod-dev.com/workspaces" target="_blank">ft-user-idf0721f5345.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [x] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

**Comment by loujaybee** [comment]
We have a community session scheduled next week on the 21st [[1](https://www.youtube.com/watch?v=Y93BcEi0fs8)]. I'd love if this were able to get merged by then, as I can ask users to share their ID's with us for inclusion in the private beta. Right now it's a painful process of having to ask someone with admin access to map a user name or email to get the ID for inclusion in private betas or feature flags. Having this in would be a big help 🙏

**Review by gtsiolis — APPROVED** [review]
Looks shippable, @filiptronicek. :shipit: 

We could later on think how we can improve the user profile layout so that resource IDs show up consistently, but this already looks like an improvement.

Approving to unblock merging. 🏁 

Looping in @gitpod-io/engineering-webapp in case code changes need a closer look although seem trivial. 🏓

File: components/dashboard/src/user-settings/Account.tsx [hunk 1]
```diff
@@ -103,6 +104,7 @@ export default function Account() {
                         }}
                         errorMessage={errorMessage}
                         emailIsReadonly={!canUpdateEmail}
+                        user={user}
```
[the remark this answers is not in the document]
Comment (filiptronicek) [reply components/dashboard/src/user-settings/Account.tsx:107]: My intuition says that the ID should always be there independent of anything else. In the typescript types, it is a required field so I think we can expect it.

[the remark this answers is not in the document]
Comment (filiptronicek) [reply components/dashboard/src/user-settings/Account.tsx:107]: We could only pass the ID to the component, but this looked more readable to me. Don't mind changing it.

**Comment by filiptronicek** [comment]
Let's :shipit: and iterate later 

/unhold your horses folks

**Comment by gtsiolis** [comment]
Thanks for taking a bias for action, @filiptronicek. ✨

**Pull request merged**